zellij-server/src/panes/plugin_pane.rs

```diff
@@ -595,4 +634,25 @@ impl PluginPane {
             self.handle_plugin_bytes(client_id, bytes.clone());
         }
     }
+    fn display_request_permission_message(&self, plugin_permission: &PluginPermission) -> String {
+        let cyan = style!(self.terminal_emulator_colors.borrow().cyan).bold();
+
+        let mut messages = String::new();
+        let permissions: HashSet<PermissionType> =
+            plugin_permission.permissions.clone().into_iter().collect();
+
+        messages.push_str(&format!(
+            "Plugin {} would like permission to:",
+            cyan.paint(&plugin_permission.name),
+        ));
```

Comment:
I have a question about this part, I want to apply a color to the message, but the correct color is not applied. (only grayed out)

Is it a problem with the way I'm getting the color? Or is it another problem?

Reply:
Yeah, there is a bit of a mess here, sorry :/

To make this work you can change it to:
```rust
         let cyan = style!(self.style.colors.cyan).bold();
```